### PR TITLE
Inventory Plot using R method

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by Lunalo** [comment]
@maxwellfundi  
It is good to upload ready changes to avoid confusing @dannyparsons

File: instat/dlgInventoryPlot.designer.vb [hunk 1]
```diff
@@ -22,148 +22,142 @@ Partial Class dlgInventoryPlot
...
-        Me.lblDry = New System.Windows.Forms.Label()
-        Me.lblWet = New System.Windows.Forms.Label()
-        Me.lblColour = New System.Windows.Forms.Label()
         Me.ucrBase = New instat.ucrButtons()
+        Me.UcrInventoryPlotSelector = New instat.ucrSelectorByDataFrameAddRemove()
+        Me.ucrSaveInventoryPlot = New instat.ucrSaveGraph()
+        Me.ucrYearReceiver = New instat.ucrReceiverSingle()
+        Me.lblYear = New System.Windows.Forms.Label()
+        Me.ucrDayOfYearReceiver = New instat.ucrReceiverSingle()
+        Me.UcrReceiverSingle2 = New instat.ucrReceiverSingle()
```
Comment (Lunalo) [inline instat/dlgInventoryPlot.designer.vb]: Problem with naming convention

**Comment by maxwellfundi** [comment]
@Lunalo  Read the title of that pull request, did you see the In Progress part? 

its also to let people know am working on that. 

@dannyparsons did you get confused by this?

File: instat/dlgInventoryPlot.designer.vb [hunk 2]
```diff
@@ -22,148 +22,142 @@ Partial Class dlgInventoryPlot
...
-        Me.lblYLabel = New System.Windows.Forms.Label()
-        Me.chkSort = New System.Windows.Forms.CheckBox()
-        Me.chkRemoveNA = New System.Windows.Forms.CheckBox()
-        Me.txtWetColour = New System.Windows.Forms.TextBox()
-        Me.txtDryColour = New System.Windows.Forms.TextBox()
-        Me.lblDry = New System.Windows.Forms.Label()
-        Me.lblWet = New System.Windows.Forms.Label()
-        Me.lblColour = New System.Windows.Forms.Label()
         Me.ucrBase = New instat.ucrButtons()
+        Me.UcrInventoryPlotSelector = New instat.ucrSelectorByDataFrameAddRemove()
```
Comment (Lunalo) [inline instat/dlgInventoryPlot.designer.vb]: Problem with naming convention

**Comment by Lunalo** [comment]
@maxwellfundi 
You dont understand. It is very effecient to create a pull request after you have finished working on something. If you want to let people know that you are working on something then open empty pull request or create an issue. I hope this helps.

**Comment by dannyparsons** [comment]
I generally don't look at pull requests until someone has reviewed them. 
However, it's good if there are consistent guidelines for doing pull requests that is agreed and understood by everyone in the team so that you work effectively and avoid confusion. What those exact guidelines are I'm less concerned about, as long as everyone understands them.

**Comment by maxwellfundi** [comment]
@Lunalo it does!

**Comment by rdstern** [comment]
Great that you are re-doing this plot in ggplot2.  I found it difficult to see the dialogue from the skype picture.  Happy for you to add it here.  It looked though as if it doesn't yet have the range of options that the previous version had.  If we just take the daily data then the "standard version" of the plot is probably for one station and one element.  So, if there are multiple stations, then perhaps this would be allowed and the station would be the facet.
Alternatively, if there are multiple stations, then "time" is along the x-axis and the stations would be the different "rows, i.e. heights up the y-axis.
Similarly there could be multiple elements (and possibly multiple stations).  Then again could have a graph for each station, with the y being the element, and different stations being either also  the y or (perhaps better) being facets.  Then time is again the x-axis.

So, in the dialogue there is a receiver for the station column and then one for the date and then a choice between a multiple receiver if there are multiple elements, or a single receiver for just one element.

It is possible that R-Instat will "know" the date and also the station columns.  In that case they can be "filled-in" once the data frame has been selected.

Exciting.

Roger

File: instat/dlgInventoryPlot.designer.vb (hunk 1 above)
Comment (maxwellfundi) [inline instat/dlgInventoryPlot.designer.vb]: Same with this.

**Comment by maxwellfundi** [comment]
@rdstern  stern, thanks for your insights. What we currently have here does not really meet some of the precise details you gave. 
It can only give an plot for day of year and years. 
Its part of our discussions with @stevekogo to enhance the dialog to accommodate all these sort of things.  
We might have a discussion together fairly soon to discuss better implementation of this. 
inventoryplot2


Thanks

**Comment by maxwellfundi** [comment]
@stevekogo @Lunalo  this is ready for review. It works well and i think it is ready  for merging. 

Thanks.

**Comment by maxwellfundi** [comment]
This branch just got a conflict. @dannyparsons  i am closing this and sending a fresh one.

**Pull request closed without merging**